data/extension/init/templates/tinygo/envoy.filters.network/default/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"github.com/tetratelabs/proxy-wasm-go-sdk/proxywasm"
+	"github.com/tetratelabs/proxy-wasm-go-sdk/proxywasm/types"
+)
+
+var (
+	connectionCounterName = "proxy_wasm_go.connection_counter"
+	counter               proxywasm.MetricCounter
+)
+
+type context struct{ proxywasm.DefaultContext }
+
+func main() {
+	proxywasm.SetNewRootContext(func(contextID uint32) proxywasm.RootContext { return context{} })
+	proxywasm.SetNewStreamContext(func(contextID uint32) proxywasm.StreamContext { return context{} })
+}
+
+func (ctx context) OnVMStart(int) bool {
```

Comment:
This API creates an impression that `OnVMStart()` and `OnNewConnection()` will be called in the same instance of `context `.

Is it actually the case ? Otherwise, it's misleading.